src/screens/SendSynthetic/SendSyntheticAmount.js

```diff
@@ -309,7 +310,10 @@ const combinedMapStateToProps = (state: RootReducerState): $Shape<Props> => ({
 });
 
 const mapDispatchToProps = (dispatch: Dispatch): $Shape<Props> => ({
-  fetchSingleAssetRates: (assetCode: string) => dispatch(fetchSingleAssetRatesAction(assetCode)),
+  fetchSingleAssetRates: (
+    chain: Chain,
+    assetCode: string,
+  ) => dispatch(fetchSingleChainAssetRatesAction(chain, assetCode)),
```

Comment:
That would be simplier ;-)
```suggestion
  ) => dispatch(fetchSingleChainAssetRatesAction(CHAIN.ETHEREUM, assetCode)),
```

Reply:
👍